plugins/hls-eval-plugin/test/Main.hs

```diff
@@ -88,6 +88,7 @@ tests =
   , goldenWithEval "Reports an error for an incorrect type with :kind" "T13" "hs"
   , goldenWithEval' "Returns a fully-instantiated type for :type" "T14" "hs" (if ghcVersion >= GHC98 then "ghc98.expected" else "expected") -- See https://gitlab.haskell.org/ghc/ghc/-/issues/24069
   , knownBrokenForGhcVersions [GHC92, GHC94, GHC96, GHC98] "type +v does not work anymore with 9.2" $ goldenWithEval "Returns an uninstantiated type for :type +v, admitting multiple whitespaces around arguments" "T15" "hs"
+  , goldenWithEval "T4139" "T4139" "hs"
```

Comment:
Looks good.
Now just mark it as known broken and give it more human readable test name, like:

```suggestion
  , knownBrokenForGhcVersions [GHC98] $ goldenWithEval "Doesn't break in module containing main function" "T4139" "hs"
```